Gopkg.lock

```diff
@@ -4,7 +4,10 @@
 [[projects]]
   branch = "master"
   name = "github.com/Azure/go-ansiterm"
-  packages = [".","winterm"]
+  packages = [
```

Comment:
seems to upgrade lot of unrelated packages. did you try `git add -p` to add only the relevant changes?

Reply:
:man_shrugging:  This is the output from `make build`, it just indented the array I guess